### ISSUE TITLE
Dependencies file with tag files and the resulting doxygen "dxy"  file

### ISSUE THREAD

**Issue by albert-github** [CGAL/cgal#8211]
With the fix b9ee8c6 a fix (by @sloriot) was made to fix the issue:
```
error: Tag file '.../doc_tags/Lab.tag' does not exist or is not a file. Skipping it...
```
when building the documentation of the (not really supported) Three  package.
The fix as is is correct and worked for a clean build, but when using an existing build it didn't work as the `Three.dxy` configuration file was not rebuild and a remake of the cmake files was not triggered.
The change was made in the file `Three/doc/Tree/dependencies` and this is not listed as a dependency to regenerate the `Three.dxy`

In an email discussion it was pointed out by @lrineau that:
> The answer is probably https://cmake.org/cmake/help/latest/prop_dir/CMAKE_CONFIGURE_DEPENDS.html

The solution lies probably in Documentation/doc/CMakeLists.txt where a configuration dependency between:
`${depend}/doc/${depend}/dependencies` and `${CGAL_DOC_DXY_DIR}/${depend}.dxy` has to be created, where `depend` is a package name).

**Comment by lrineau** [comment]
I was bitten by that bug, and it made me loss minutes of work. So I fixed it in the unrelated PR #8248 (commit 50d7abaec581981930f8093a8f0668a83e6406c8).

**Comment by lrineau** [comment]
> I was bitten by that bug, and it made me loss minutes of work. So I fixed it in the unrelated PR #8248 (commit [50d7aba](https://github.com/CGAL/cgal/commit/50d7abaec581981930f8093a8f0668a83e6406c8)).

Should be fixed in CGAL `master`, now. @albert-github, do you confirm the bug is fixed?

**Comment by albert-github** [comment]
I did a quick test.
- build the documentation
- change the dependencies and build the documentation 
   - CMake was automatically rerun
   - the new dependencies were present 
- undo the change inthe dependencies and build the documentation 
   - CMake was automatically rerun
   - the old dependencies were present 

This confirms to me that this problem is fixed.
